src/components/OtherCitiesList.jsx

```diff
@@ -0,0 +1,76 @@
+import { useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import { addClickedCity } from 'redux/home/homeSlice';
+
+const OtherCitiesList = ({ otherCities, isLoading, error }) => {
+  // const { otherCities, isLoading, error } = useSelector(getData);
```

Comment:
- Please, kindly remove these commented pieces of code to make your codebase clean and maintainable